synfig-studio/src/gui/app.cpp

```diff
@@ -1349,12 +1350,15 @@ App::get_default_accel_map()
 
 /* === M E T H O D S ======================================================= */
 
-App::App(const synfig::String& basepath, int *argc, char ***argv):
-	Gtk::Main(argc,argv)
+Glib::RefPtr<App> App::create()
 {
+	return Glib::RefPtr<App>(new App());
+}
 
+void App::on_startup()
+{
+	
 	Glib::init(); // need to use Gio functions before app is started
-	app_base_path_=etl::dirname(basepath);
```

Comment:
Why did you remove it?
This is probably the crash reason.

Reply:
Okkay ! I thought I will fix this part later ! NVM will add it right away ! 🙂